### ISSUE TITLE
`-a` downloading is broken

### ISSUE THREAD

**Issue by Faiz7412** [flyingrub/scdl#130]
Hi, I was trying to install scdl. It threw an error:"`permission denied`  
```
running install_lib

creating /usr/local/lib/python3.4/dist-packages/scdl

error: could not create '/usr/local/lib/python3.4/dist-packages/scdl': Permission denied

----------------------------------------
Cleaning up...
Command /usr/bin/python3 -c "import setuptools, tokenize;__file__='/tmp/pip_build_faiz/scdl/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-si9lwwcl-record/install-record.txt --single-version-externally-managed --compile failed with error code 1 in /tmp/pip_build_faiz/scdl
Storing debug log for failure in /home/faiz/.pip/pip.log
```
So I went ahead and install with `sudo pip3 install scdl`  
I got the following error while trying to download tracks from [Mr. Suicidesheep](https://soundcloud.com/mrsuicidesheep) with `scdl -l https://soundcloud.com/mrsuicidesheep -a`: 
```
Track n°811 of 836
'streamable'
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/scdl/scdl.py", line 262, in download
    download_track(item)
  File "/usr/local/lib/python3.4/dist-packages/scdl/scdl.py", line 346, in download_track
    if track['streamable']:
KeyError: 'streamable'
```
It iterated through the entire 836 entries, giving the same error for all tracks except one which it was able to download.   I ran it again:
```
faiz@fizz:~$ scdl -l https://soundcloud.com/mrsuicidesheep -a -c
Soundcloud Downloader

Starting new HTTPS connection (1): api.soundcloud.com
Found a user profile
Retrieving all the tracks and reposts of user MrSuicideSheep...
Starting new HTTPS connection (1): api-v2.soundcloud.com
Starting new HTTPS connection (1): api-v2.soundcloud.com
Starting new HTTPS connection (1): api-v2.soundcloud.com
Starting new HTTPS connection (1): api-v2.soundcloud.com
Starting new HTTPS connection (1): api-v2.soundcloud.com
Retrieved 837 tracks and reposts
Tracks and reposts n°1 of 837
Starting new HTTPS connection (1): api.soundcloud.com
Found a track
Starting new HTTPS connection (1): api.soundcloud.com
Downloading Noble Oak - Dream Spark
Noble Oak - Dream Spark already Downloaded

Track n°2 of 837
'streamable'
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/scdl/scdl.py", line 262, in download
    download_track(item)
  File "/usr/local/lib/python3.4/dist-packages/scdl/scdl.py", line 346, in download_track
    if track['streamable']:
KeyError: 'streamable'
Track n°3 of 837
'streamable'
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/scdl/scdl.py", line 262, in download
    download_track(item)
  File "/usr/local/lib/python3.4/dist-packages/scdl/scdl.py", line 346, in download_track
    if track['streamable']:
KeyError: 'streamable'
//similar errors for all the entries
//at the end I got this
Track n°837 of 837
'streamable'
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/scdl/scdl.py", line 262, in download
    download_track(item)
  File "/usr/local/lib/python3.4/dist-packages/scdl/scdl.py", line 346, in download_track
    if track['streamable']:
KeyError: 'streamable'
Downloaded all 837 track of user MrSuicideSheep!
```

At the end of the log file, I found this: 
```
Exception information:
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/pip/basecommand.py", line 122, in main
    status = self.run(options, args)
  File "/usr/lib/python3/dist-packages/pip/commands/install.py", line 283, in run
    requirement_set.install(install_options, global_options, root=options.root_path)
  File "/usr/lib/python3/dist-packages/pip/req.py", line 1436, in install
    requirement.install(install_options, global_options, *args, **kwargs)
  File "/usr/lib/python3/dist-packages/pip/req.py", line 707, in install
    cwd=self.source_dir, filter_stdout=self._filter_install, show_stdout=False)
  File "/usr/lib/python3/dist-packages/pip/util.py", line 715, in call_subprocess
    % (command_desc, proc.returncode, cwd))
pip.exceptions.InstallationError: Command /usr/bin/python3 -c "import setuptools, tokenize;__file__='/tmp/pip_build_faiz/scdl/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-si9lwwcl-record/install-record.txt --single-version-externally-managed --compile failed with error code 1 in /tmp/pip_build_faiz/scdl
```

**Comment by flyingrub** [comment]
can you run `scdl --version` ?

**Comment by Faiz7412** [comment]
``` faiz@fizz:~$ scdl --version
v1.5.0-3
```

**Comment by flyingrub** [comment]
mmh soundcloud changed their API.

**Comment by Faiz7412** [comment]
Perhaps some small tweaks because I was able to download one song.

**Comment by flyingrub** [comment]
yup i already know the fix, just don't have the time to push right now.

**Comment by Faiz7412** [comment]
Okay.  I'll try to go through the code if I have time and see if I can fix it.  Anyway, by when can we expect the fix?

**Comment by andrw85** [comment]
Hi,

I'm getting the same error as well. It just downloads the first track after that it just keep saying the same error when trying to download each of the tracks of a playlist:

scdl -l http://soundcloud.com/maxzwell/ -a
Soundcloud Downloader

Starting new HTTPS connection (1): api.soundcloud.com
Found a user profile
Retrieving all the tracks and reposts of user maxzwell...
Starting new HTTPS connection (1): api-v2.soundcloud.com
Retrieved 48 tracks and reposts
Tracks and reposts n°1 of 48
Starting new HTTPS connection (1): api.soundcloud.com
Found a track
Starting new HTTPS connection (1): api.soundcloud.com
Downloading Tausender Beträge (prod. Maxzwell) - Svaba Ortak ft The Ji
Starting new HTTPS connection (1): cf-media.sndcdn.com
[################################] 2709/2709 - 00:00:01
Settings tags...
Starting new HTTPS connection (1): api.soundcloud.com
Starting new HTTPS connection (1): i1.sndcdn.com

Tausender Beträge (prod. Maxzwell) - Svaba Ortak ft The Ji.mp3 Downloaded.

Track n°2 of 48
'streamable'
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/scdl/scdl.py", line 262, in download
    download_track(item)
  File "/usr/local/lib/python3.4/dist-packages/scdl/scdl.py", line 346, in download_track
    if track['streamable']:
KeyError: 'streamable'
Track n°3 of 48
'streamable'
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/scdl/scdl.py", line 262, in download
    download_track(item)
  File "/usr/local/lib/python3.4/dist-packages/scdl/scdl.py", line 346, in download_track
    if track['streamable']:
KeyError: 'streamable'
Track n°4 of 48
'streamable'


scdl --version
v1.5.0-3

Any fix available for this?

Thanks!

**Comment by Clevero** [comment]
I would also be glad if anyone can fix this :+1:

**Comment by nablaa** [comment]
Fix for this is in pull request #140.
